app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
It's not a good practice to use mutable class variables unless it's really necessary. This can lead to unexpected behavior and bugs because all instances of the class share the same variable. Consider moving `people` dictionary to a more appropriate place, for example, to the `create_person_list` function or another class that is responsible for managing people.